### ISSUE TITLE
After restart redmine

### ISSUE THREAD

**Issue by adrianobr** [a-ono/redmine_ckeditor#318]
![image](https://user-images.githubusercontent.com/7975205/83695204-fd050780-a5cf-11ea-9cd0-8bea007fd550.png)

can help?

**Comment by chfrcodan** [comment]
Did you try removing the Cache directory as suggested? Or checking (and perhaps overwriting) permissions on /usr/local/bundle/cache/bundler/git/rich-......??

**Comment by adrianobr** [comment]
Yes I did it. I have the redmine on an EC2 AWS in container, and every time the container goes up it must connect the plugins. From what I noticed he can't clone the repository right.
So at Gemfile I changed the call to this:
gem 'rich', path: '/ usr / src / redmine / dependencies / rich'

And in the directory I did git clone manually.
I didn't understand why it didn't work as you have ... But switching to the path and downloading manually via git clone worked for me.

**Comment by adrianobr** [comment]
Would it be this?
git rm --cached .

**Comment by veshus** [comment]
I have the same issue with an official Redmine docker container. After docker restart - it's dying.

**Comment by adrianobr** [comment]
I solved:
chmod -R a+w /usr/src/redmine/public/system/
